sphere-models/src/it/java/io/sphere/sdk/products/search/ProductProjectionSearchMainIntegrationTest.java

```diff
@@ -64,6 +76,79 @@ public void fuzzySearch() {
         assertThat(execute(search.withFuzzy(true)).getResults()).matches(containsIdentifiable(product2), "included");
     }
 
+    @Test
+    public void findMatchingVariantByQueryFilter() throws Exception {
+        testBlueFilter(filters -> ProductProjectionSearch.ofStaged().withQueryFilters(filters));
+    }
+
+    @Test
+    public void findMatchingVariantByResultFilter() throws Exception {
+        testBlueFilter(filters -> ProductProjectionSearch.ofStaged().withResultFilters(filters));
+    }
+
+    @Ignore // HTTP API still not having sort associated to matching variant flag
+    @Test
+    public void findMatchingVariantBySort() throws Exception {
+        final ProductProjectionSearch search = ProductProjectionSearch.ofStaged()
+                .withSort(product -> product.allVariants().attribute().ofNumber(ATTR_NAME_SIZE).byAscWithMax());
+        assertThat(executeSearch(search).getResults()).are(productWithMatchingVariantsHavingMaxSize());
+    }
+
+    @Ignore // HTTP API doesn't mark the correct variant as true when the searched info is inside the variant
```

Comment:
Can you open a GitHub issue to memorize to reenable the tests and link the issue here?